src/dfx-core/src/cli/mod.rs

```diff
@@ -0,0 +1,16 @@
+use std::io::stdin;
+
+pub fn ask_for_consent(message: &str) -> Result<(), String> {
```

Comment:
Let's use concrete error types in dfx-core, not String.  See for example `EncryptionError::ReadUserPasswordFailed(std::io::Error)`.